packages/react/src/lib/accessibility/FocusZone/CHANGELOG.md

```diff
@@ -26,6 +26,12 @@ This is a list of changes made to this Stardust copy of FocusZone in comparison
 - Handle keyDownCapture based on `shouldHandleKeyDownCapture` prop @sophieH29 ([#563](https://github.com/stardust-ui/react/pull/563))
 - Add `bidirectionalDomOrder` direction allowing arrow keys navigation following DOM order @sophieH29 ([#1637](https://github.com/stardust-ui/react/pull/1647))
 
+### Upgrade `FocusZone` to the latest version from `fabric-ui` @sophieH29 ([#1772](https://github.com/stardust-ui/react/pull/1772))
```

Comment:
Does it also include OfficeDev/office-ui-fabric-react#8421?

Reply:
yes, it does. missed it, thx